### DISCUSSION TITLE
How to send email via port 587

### DISCUSSION THREAD

**Discussion by rafale0n** [emersion/maddy#490] [Q&A]
Hello, thanks for maddy!

I was curious how to switch from port 25 to 587 as default to send email with maddy ?

Cheers!

**Comment by foxcpp** [comment]
There is no such option as 587 port is not meant to be used for outbound email delivery. If your ISP blocks port 25 you should send messages by relaying them via an external host that has port 25 unblocked, sending via 587 will not work in most cases.
